esi_ui/static/dashboard/esi/lessee/nodes/esi-nodes-table.controller.js

```diff
@@ -364,6 +368,35 @@
         }
       });
     }
+    function getLeaseColor(endDate) {
+      const now = new Date();
+      const end = new Date(endDate);
+      const diff = end - now;
+    
+      if (diff <= 7 * 24 * 60 * 60 * 1000) {
+        return 'red';
+      } else if (diff <= 30 * 24 * 60 * 60 * 1000) { 
+        return 'orange';
+      } else {
+        return 'green';
+      }
+    }
+
+    function getProvisionStateColor(provisionState, targetProvisionState) {
+      if (targetProvisionState) {
+        return 'blue';
+      }
+      if (PROVISION_ERROR_STATES.has(provisionState)) {
+        return 'red'; 
+      }
+      if (PROVISION_STABLE_STATES.has(provisionState)) {
+        return provisionState === 'active' ? 'green' : 'brown';
```

Comment:
Can we change this brown to also blue? The brown looks pretty alarming, making it look like there might be something wrong with the node.

Reply:
Even I felt that way. I tried differnt color but didn't suit. So I kept it brown in the end. Changed to blue now.